tailcfg/tailcfg.go

```diff
@@ -1429,6 +1451,11 @@ const (
 	// user groups as Kubernetes user groups. This capability is read by
 	// peers that are Tailscale Kubernetes operator instances.
 	PeerCapabilityKubernetes PeerCapability = "tailscale.com/cap/kubernetes"
+
+	// PeerCapabilityServicesDestination grants a peer the ability to serve as
+	// a destination for a set of given VIP services, which is provided as the
```

Comment:
reference the Go type (perhaps with the new fancy `[.....]` godoc syntax) that the value is of?

Reply:
oops, I just missed writing the type `PeerCapability` in the declaration, I can add that in there